pgctl/cli.py

```diff
@@ -460,18 +475,19 @@ def _expand_aliases(self, name):
         return result
 
     @cached_property
-    def all_service_names(self):
-        """Return a tuple of all of the Services.
+    def all_services(self):
+        """Return a list of all services.
 
-        :return: tuple of strings -- the service names
+        :return: list of Service objects
+        :rtype: list
         """
         pgdir = self.pgdir.listdir(sort=True)
 
-        return tuple(
-            service_path.basename
+        return [
```

Comment:
i'd probably keep as a tuple since this is a cached property and ideally would be immutable